css/style.css

```diff
@@ -556,11 +556,11 @@ figcaption details p {
     margin-bottom: 2em;
 }
 
-.floe .floe-image-UIO             { background: url("../images/floe-02.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-exploreTool     { background: url("../images/floe-01.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-cloud           { background: url("../images/floe-05.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-preferenceIcons { background: url("../images/floe-04.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-videoPlayer     { background: url("../images/floe-03.png") no-repeat 0 0 / 325px 325px; }
+.floe .floe-image-UIO             { background: url("../images/floe-02.png") no-repeat 0 0 / 290px 290px; }
```

Comment:
The issue is being caused by the fact that the div containing the figure and the parent div container has different width values which are defined in pixels and not because of size of photo in pixels.. the div containing the figure as background has dimensions bigger than the parent container. Fixing the div container classes and using percentage values for width and setting the max-width property will solve this. Explicitly changing the size of images doesn't seem to be the correct solution for this. 
It causes the following problems:
1) Images are not aligned to the center for small screen devices.
![Screenshot from 2020-03-03 09-41-25](https://user-images.githubusercontent.com/34926285/75742383-6fe1b680-5d33-11ea-992e-cc1fbcf0dea3.png)
2) The Images do not scale themselves in accordance with the width of the body tag.
![Screenshot from 2020-03-03 09-42-35](https://user-images.githubusercontent.com/34926285/75742453-a9b2bd00-5d33-11ea-9f91-73c12ca20530.png)

Reply:
Yeah, right!